Sources/Styleguide/PointFreeBaseStyles.swift

```diff
@@ -348,6 +350,7 @@ private let colorStyles: Stylesheet =
   Class.pf.colors.bg.black % backgroundColor(Colors.black)
     <> Class.pf.colors.bg.blue900 % backgroundColor(Colors.blue900)
     <> Class.pf.colors.bg.dark % backgroundColor(Colors.black)
+    <> Class.pf.colors.bg.gray150 % backgroundColor(Colors.gray150)
```

Comment:
new atomic style for parking backgrounds, will cause lots of snapshots to update